### ISSUE TITLE
Unblurr images on Patreon and SubscribeStar

### ISSUE THREAD

**Issue by golgon1995** [Simon-Tesla/RaccoonyWebEx#178]
Can you please give Raccoony the ability to unblurr images on Patreon and SubscribeStar?

**Comment by Simon-Tesla** [comment]
I'm not sure I understand what you're asking for specifically.

**Comment by golgon1995** [comment]
> I'm not sure I understand what you're asking for specifically.


Do THESE ring a bell to you?:

https://www.patreon.com/MAJUV

https://subscribestar.adult/paddedulf

https://pieceofsoap.fanbox.cc/posts

**Comment by Simon-Tesla** [comment]
Ah, the blur you get when you're either not logged in or not a subscriber, I see. Gonna wont-fix this for several reasons:

- This isn't that kind of extension. I have no interest in writing anything to help you steal content.
- Even if I did, Patreon and other sites blur these images destructively. It isn't possible to get anything resembling the original data back out of them if you have a blurred image.
- Raccoony just downloads picture URLs. It does not look at or modify image data itself.

In conclusion, kindly fuck off.

**Comment by Simon-Tesla** [comment]
wrong close reason
